src/platform/forms/tests/save-in-progress/actions.unit.spec.js

```diff
@@ -320,6 +335,27 @@ describe('Schemaform save / load actions:', () => {
         );
       });
     });
+    it('dispatches a success if the form is loaded', () => {
+      const thunk = fetchInProgressForm(VA_FORM_IDS.FORM_21_526EZ, {});
+      const dispatch = sinon.spy();
+      global.fetch.returns(
+        Promise.resolve({
+          ok: true,
+          json: () => ({
+            formData: { field: 'foo' }, // eslint-disable-line camelcase
```

Comment:
Is this `eslint-disable-line` needed?

Reply:
I don't think so... did I add that? 😱

Reply:
LOL, I copy/pasted it from the test above 😛